src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -59,28 +59,34 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
   }
 
   /**
-   * This function fires when a user submits the login form. It gets the username from the submitted
-   * form data, and then adds it to the session so we know the user is logged in.
+   * This function fires when a user submits the login form. It gets the username and password from the submitted
+   * form data, checks that they're valid, and either adds the user to the session so we know the user is logged in.
```

Comment:
Extraneous period

Reply:
I'm unsure of what you meant, could you clarify?

Reply:
Just a minor typo in your commenting "user is logged in. or shows an error"

Reply:
Oh okay. Got it!